quicktests/overlaying/tests/basic/wheel.js

```diff
@@ -0,0 +1,35 @@
+function makeData() {
+  "use strict";
+
+  return [
+    { id: "0", r1: 0, r2: 1, t1: -60, t2: 60 },
+    { id: "1", r1: 0, r2: 1, t1: 180, t2: 300 },
+    { id: "2", r1: 0, r2: 1, t1: 60, t2: 180 },
+    { id: "3", r1: 1, r2: 2, t1: -60, t2: 60 },
+    { id: "4", r1: 1, r2: 2, t1: 180, t2: 300 },
+    { id: "5", r1: 1, r2: 2, t1: 60, t2: 180 },
+    { id: "6", r1: 2, r2: 3, t1: -60, t2: 60 },
+    { id: "7", r1: 2, r2: 3, t1: 180, t2: 300 },
+    { id: "8", r1: 2, r2: 3, t1: 60, t2: 180 },
+    { id: "9", r1: 3, r2: 4, t1: 60, t2: 180 }
```

Comment:
Maybe pick some data that shows off the ability to create non-complete (and/or non-touching) arcs? Right now the plot looks like this:

<img width="535" alt="screen shot 2015-08-31 at 11 14 15 am" src="https://cloud.githubusercontent.com/assets/1440449/9586390/70357758-4fd1-11e5-89b1-b19c1a636e5d.png">